src/index.html

```diff
@@ -1,22 +1,315 @@
+                      src="images/details/range-and-integrated-lights-1.png"
+                      alt="Range & Integrated lights"
+                      class="feature__photo"
+                    >
+                  </a>
+
+                  <a href="#" class="feature__link feature__link--square">
+                    <img
+                    src="images/details/range-and-integrated-lights-2.png"
+                      alt="Range & Integrated lights"
+                      class="feature__photo"
+                    >
+                  </a>
+                </div>
+
+                <h3 class="feature__title">Range & Integrated lights</h3>
+                <p class="feature__description">The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.</p>
+              </article>
+
+              <article class="details__feature feature">
+                <div class="feature__photos">
+                  <a href="#" class="feature__link feature__link--square">
+                    <img
+                      src="images/details/hydraulic-disc-brakes-and-lightweight-1.png"
+                      alt="Hydraulic disc brakes & Lightweight"
+                      class="feature__photo"
+                    >
+                  </a>
+
+                  <a href="#" class="feature__link feature__link--wide">
+                    <img
+                    src="images/details/hydraulic-disc-brakes-and-lightweight-2.png"
+                      alt="Hydraulic disc brakes & Lightweight"
+                      class="feature__photo"
+                    >
+                  </a>
+                </div>
+
+                <h3 class="feature__title">Hydraulic disc brakes & Lightweight</h3>
+                <p class="feature__description">Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.</p>
+              </article>
+
+              <button class="details__button button" onclick="location.href='#'">Explore</button>
+            </div>
+          </section>
+
+          <section class="main__contact-us contact-us" id="contact-us">
+            <h2 class="main__section-title contact-us__title">
+              Contact us
+            </h2>
+
+            <div class="contact-us__wrapper">
+              <form onsubmit="this.reset(); return false;"
+                action="#"
+                method="get"
+                class="contact-us__form form"
+              >
+                <div class="form__inputs">
+                  <input
+                    type="text"
+                    minlength="3"
+                    maxlength="50"
+                    name="name"
+                    class="form__input"
+                    placeholder="Name"
+                    pattern="[A-Z][a-z]+"
```

Comment:
[Not critical]
I don't think we need to limit minimal and maximal length for a name, as well as require it to match pattern.
Usually, if there's such kind of restrictions, user should be explicitly informed about that: with placeholder, tooltip, help text etc.
This is not critical for a current task, but it'd be nice to mind that during the future work.